boards/arm/thingy91_nrf9160/dts/bindings/nordic,thingy91-nrf52840-reset.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA
```

Comment:
Maybe we could create some generic bindings to reset Soc in case if we have more Soc than one on single DK ?

Reply:
That sounds like a possibility. Do you have any input on that, @lemrey ?

Reply:
Coming back to this: I don't think it will be very productive to put a lot of effort into making this a generic thing. It's a quite narrow use case, and the customers that requires it for their own board will likely just base their board files on Thingy:91 or the nRF9160 DK.

Trying to make this more generic could just lead to confusion. I feel like the current implementation makes the functionality very clear and understandable.